libsplinter/src/admin/store/diesel/operations/add_event.rs

```diff
@@ -21,13 +21,15 @@ use diesel::{dsl::insert_into, prelude::*};
 use super::AdminServiceStoreOperations;
 
 use crate::admin::service::messages;
+#[cfg(feature = "sqlite")]
```

Comment:
Should the two implementations here actually differ?

Reply:
They do, in how the event is initially added, then the event_id is retrieved. If you look at how the postgres impl is getting the `event_id`: https://github.com/Cargill/splinter/blob/main/libsplinter/src/admin/store/diesel/operations/add_event.rs#L63, versus the sqlite impl: https://github.com/Cargill/splinter/blob/main/libsplinter/src/admin/store/diesel/operations/add_event.rs#L155

Reply:
Also, generally the `add_*` methods can't use the generic connection, if you look at the other add operations implemented for the admin store. (Most have separate implementations for the postgres and sqlite feature, though the implementations don't differ greatly)